* scikit-learn #gatesgarth #yocto
@ 2021-09-17 12:31 abhi
  0 siblings, 0 replies; only message in thread
From: abhi @ 2021-09-17 12:31 UTC (permalink / raw)
  To: yocto

[-- Attachment #1: Type: text/plain, Size: 2023 bytes --]

hi guys,

i am trying to build scikit-learn on yocto but i am getting error of setup.py ,it is showing failed i build all dependencies but it is giving error please try to give some i dea

ERROR: python3-sklearn-1.0-r0 do_compile: 'python3 setup.py build ' execution failed.
ERROR: python3-sklearn-1.0-r0 do_compile: Execution of '/home/abhishek//sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/temp/run.do_compile.17835' failed with exit code 1:
/home/abhishek//sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/recipe-sysroot-native/usr/bin/python3-native/python3: can't open file '/home/abhishek/sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/scikit-learn-1.0/setup.py': [Errno 2] No such file or directory
WARNING: exit code 1 from a shell command.

ERROR: Logfile of failure stored in: /home/abhishek/exaleap/sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/temp/log.do_compile.17835
Log data follows:
| DEBUG: Executing shell function do_compile
| /home/abhishek//sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/recipe-sysroot-native/usr/bin/python3-native/python3: can't open file '/home/abhishek//sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/scikit-learn-1.0/setup.py': [Errno 2] No such file or directory
| ERROR: 'python3 setup.py build ' execution failed.
| WARNING: exit code 1 from a shell command.
| ERROR: Execution of '/home/abhishek//sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/temp/run.do_compile.17835' failed with exit code 1:
| /home/abhishek//sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/recipe-sysroot-native/usr/bin/python3-native/python3: can't open file '/home/abhishek//sources/fu540-build/tmp-glibc/work/riscv64-oe-linux/python3-sklearn/1.0-r0/scikit-learn-1.0/setup.py': [Errno 2] No such file or directory
| WARNING: exit code 1 from a shell command.

[-- Attachment #2: Type: text/html, Size: 2091 bytes --]

^ permalink raw reply	[flat|nested] only message in thread
only message in thread, other threads:[~2021-09-17 12:31 UTC | newest]

Thread overview: (only message) (download: mbox.gz / follow: Atom feed)
-- links below jump to the message on this page --
2021-09-17 12:31 scikit-learn #gatesgarth #yocto abhi

